flakeight.sh

```diff
@@ -7,13 +7,5 @@
 #
 #   tox -epep8 -- -HEAD
 #
-
-if test "x$1" = "x-HEAD" ; then
-    shift
-    files=$(git diff --name-only HEAD~1 | tr '\n' ' ')
-    echo "Running flake8 on ${files}"
-    diff -u --from-file /dev/null ${files} | flake8 --diff "$@"
-else
-    echo "Running flake8 on all files"
-    exec python3 -m flake8 "$@"
-fi
\ No newline at end of file
+echo "Running flake8 on flocx_market"
+exec python3 -m flake8 "flocx_market"
```

Comment:
I'm not sure I understand the purpose of this change; the existing script was useful because it permitted running flake8 on only the files in the current commit.  For targeting the `flocx_market` directory specifically by default, see https://github.com/CCI-MOC/flocx-market/pull/23.

I would prefer to leave this file unmodified in this pull request.

Reply:
Ok I'll just move the virtual env file elsewhere so flake8 doesn't throw a bunch of errors for it

Reply:
If you rebase on `master` then flake8 will only run on the `flocx_market` directory by default (#23 has merged).